src/oc/web/components/entry_edit.cljs

```diff
@@ -516,6 +522,10 @@
                          :position "top"
                          :default-field-selector "div.entry-edit-modal div.rich-body-editor"
                          :container-selector "div.entry-edit-modal"})
+          (when (= true (:auto-saving entry-editing))
```

Comment:
Do we really need to check if it's equal to `true`? If so you can use `true?` that is a big more clean.
https://clojuredocs.org/clojure.core/true_q